src/scripts/lint/.eslintrc.react.js

```diff
@@ -29,10 +22,39 @@ module.exports = {
     },
   },
   rules: {
-    // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
-    // e.g. "@typescript-eslint/explicit-function-return-type": "off",
-    // "prettier/prettier": ["error", { "trailingComma": "none" }]
     "@typescript-eslint/interface-name-prefix": "off",
     "@typescript-eslint/no-empty-function": ["error", { allow: ["arrowFunctions"] }],
+    "react/sort-comp": ["error", {
+      "order": [
+        "type-annotations",
+        "static-variables",
+        "static-methods",
+        "getters",
+        "setters",
+        "lifecycle",
+        "everything-else",
+        "render",
+      ],
+    }],
+    'react/jsx-props-no-spreading': 'off',
+    'react/prop-types': 'off',
+    "arrow-parens": ["error", "as-needed", { "requireForBlockBody": true }],
```

Comment:
All of these customizations are somewhat opinionated. 

Are all going to be fixed automatically? (especially `max-len`)
Shouldn't we document them in the guidelines?